heron/common/src/java/com/twitter/heron/common/basics/FileUtils.java

```diff
@@ -121,6 +121,15 @@ public static boolean deleteDir(String dir) {
   }
 
   public static boolean deleteDir(File dir, boolean deleteSelf) {
+    if (Files.isSymbolicLink(dir.toPath())) {
```

Comment:
will this symbolic file deletion work with directories?

Reply:
This will only delete the symlink and not any of the potential files/directories under it.